master/internal/db/postgres_test_utils.go

```diff
@@ -47,6 +47,8 @@ const (
 	DefaultTestSrcPath = "../../../examples/tutorials/mnist_pytorch"
 	// DefaultProjectID is the project used when none is specified.
 	DefaultProjectID = 1
+
+	localPostgresURL = "postgres://postgres:postgres@localhost:5432/determined?sslmode=disable"
```

Comment:
I like this change, but it deserves either an explicit callout in the PR description or to be separated into another PR

Reply:
yeah, i'll pull this isn't a separate pr.